ingestion/src/metadata/sdk/data_quality/dataframes/validation_results.py

```diff
@@ -0,0 +1,91 @@
+#  Copyright 2025 Collate
+#  Licensed under the Collate Community License, Version 1.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/LICENSE
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""DataFrame validation result models."""
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+from metadata.generated.schema.tests.basic import TestCaseResult, TestCaseStatus
+
+
+@dataclass
```

Comment:
Let;s not use data class. We should use pedantic. Why not use the existing TestCaseResult object?